mappings/net/minecraft/client/texture/NativeImage.mapping

```diff
@@ -1,15 +1,57 @@
 CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
 	CLASS class_1012 Format
 		FIELD field_4993 pixelDataFormat I
-		FIELD field_4994 bytesPerPixel I
+		FIELD field_4994 totalChannels I
+		FIELD field_4996 writeable Z
+		FIELD field_4999 alpha Z
+		FIELD field_5000 luminance Z
```

Comment:
These are booleans. I think `hasAlpha` and `hasLuminance` would be less confusing.

Reply:
Fynny thing. I was just reading the CONVENTIONS.md last night, and it specifically said _not_ to do what you just suggested.

Reply:
> has prefixes **when possible** (colored, not isColored or hasColor).

The word "color" has an adjective form (colored), so we can avoid "has" here. This is not the case for "luminance". For "alpha", "translucent" could work instead of "hasAlpha".

Reply:
`hasAlpha` and `hasLuminance` are more descriptive without being verbose, because there's no adjective form of `alpha` and `luminance` like there is `colored`.